gapic-generator/templates/default/package/_paths.erb

```diff
@@ -0,0 +1,8 @@
+<%- assert_locals package -%>
+# Path helper methods for the <%= package.name %> API.
+module Paths
+<%- package.references.each do |resource| -%>
+<%= indent render(partial: "package/resource", locals: { resource: resource }), 2 %>
+
+<%- end %>
+end
```

Comment:
Might be a good idea to `extend self` so that helpers can be invoked as `Google::Cloud::WhateverService::V1::Paths.my_path`

(Alternatively, the original design had the client class extend Paths as well. However, I don't think we need to do both, and if we have a choice, I prefer these helper showing up as module methods on Paths rather than the client class. One reason is, Paths is now the union of all helpers relevant to all clients, so each client may use only a subset of them.)